data/presets/shop/_vacant.json

```diff
@@ -1,5 +1,5 @@
 {
-    "icon": "maki-shop",
+    "icon": "fas-store-alt-slash",
```

Comment:
was there a reason not to use the default name of the icon `shop-slash`?

Reply:
No. I copied the name from the disused-preset.